frontend/src/redux/TagsIgnoredReducer.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction, CaseReducer } from '@reduxjs/toolkit';
+
+import { TagName } from 'models/Tag';
+import TagsIgnored from 'models/TagsIgnored';
+
+type TagsIgnoredState = TagsIgnored;
+
+type ReceivedTagsIgnoredAction = PayloadAction<TagsIgnored>;
+type TagIgnoreToggledAction = PayloadAction<TagName>;
+
+const setTagsIgnored: CaseReducer<TagsIgnoredState, ReceivedTagsIgnoredAction> = (state, action) => {
+  return action.payload;
+};
+
+const toggleTagIgnore: CaseReducer<TagsIgnoredState, TagIgnoreToggledAction> = (state, action) => {
+  const tagName = action.payload;
+  state[tagName] = !state[tagName];
+  return state;
```

Comment:
I dont think it actually matter but you dont need to return state. It will modify it in place

Reply:
Ok yep, will take a look in the next branch.